build_ql_masters

```diff
@@ -0,0 +1,202 @@
+                    else pargs.redux_dir
+
+    # Set the output dir where the masters should be placed.
+    if pargs.output_dir is None:
+        if 'QL_MASTERS' in os.environ:
+            ql_masters = os.environ['QL_MASTERS']
+        else:
+            ql_masters = 'QL_MASTERS'
+    else:
+        ql_masters = pargs.output_dir
+    
+    # Build the list of files to copy and where to copy them to.
+    # This includes updating any filenames to match what the quicklook
+    # scripts expect, and may include generating the files uisng the dev suite.
+    source_files = []
+    dest_files = []
+
+    if pargs.instrument == 'all' or pargs.instrument == 'keck_nires':
+        if pargs.setup == 'all' or pargs.setup =='NIRES':
+
+            nires_dest_dir = os.path.join(ql_masters, 'NIRES_MASTERS')
+            os.makedirs(nires_dest_dir, exist_ok=True)
+
+            keck_nires_path = os.path.join(redux_dir, 'keck_nires', 'NIRES', 'Masters')
+            source_filters = [os.path.join(keck_nires_path, '*2_01*'),
+                            os.path.join(keck_nires_path, 'MasterFlat_A_7_01.fits'), 
+                            os.path.join(keck_nires_path, 'MasterSlits_A_7_01.fits.gz')]
+
+            if not pargs.force_build:
+                found_files = find_source_files(source_filters)
+            else:
+                found_files = []
+
+            if len(found_files) == 0:
+                print ("Could not find all NIRES masters, attempting to generate them...")
+                run_devsuite('keck_nires', 'NIRES', redux_dir)
+
+                found_files = find_source_files(source_filters)
+                if len(found_files) == 0:
+                    print ("Could not generate all NIRES masters.")
+                    sys.exit(1)
+
+
+            source_files += found_files
+            for file in found_files:
+                new_base = os.path.basename(file).replace('_2_','_1_').replace('_7_','_1_')
+                dest_files.append(os.path.join(nires_dest_dir, new_base))
+
+    if pargs.instrument == 'all' or pargs.instrument == 'keck_mosfire':
+
+        if pargs.setup == 'all' or pargs.setup == "Y_long":
+            dest_dir = os.path.join(ql_masters, 'MOSFIRE_MASTERS', 'Y')
+            os.makedirs(dest_dir, exist_ok=True)
+
+            files = ['Masters/MasterSlits_A_15_01.fits.gz',
+                     'Masters/MasterTilts_A_4_01.fits', 
+                     'Masters/MasterWaveCalib_A_4_01.fits',
+                     'Science/spec1d_m191118_0064-GD71_MOSFIRE_2019Nov18T104704.507.fits',
+                     'sens_m191118_0064-GD71_MOSFIRE_2019Nov18T104704.507.fits' ]
+
+            source_filters = [os.path.join(redux_dir, 'keck_mosfire', 'Y_long', x) for x in files]
+            
+            
+            if not pargs.force_build:
+                found_files = find_source_files(source_filters)
+            else:
+                found_files = []
+
+            if len(found_files) == 0:
```

Comment:
Does this also run the afterburner for mosfire? Because mosfire_ql requires pypeit_sensfunc be run which is now part of the afterburner, but it was not clear to me whether run_devsuite was just running the reduction or both reduction and afterburner. 

We probably want that to be an option in run_devsuite, since there may be situations for which we don't need to run the afterburner to generate QL masters. Thoughts?